viadot/sources/cloud_for_customers.py

```diff
@@ -90,7 +90,6 @@ def _to_records_report(self, url: str) -> List[Dict[str, Any]]:
             records.extend(new_records)
 
             url = response_json["d"].get("__next")
```

Comment:
Please consider to use solution from this code:
https://github.com/adrian-wojcik/viadot/blob/a62174515cf0326d1d15e32f151b609522d87b21/viadot/sources/cloud_for_customers.py